.github/workflows/build.yml

```diff
@@ -0,0 +1,49 @@
+name: Build
+
+on:
+  push:
+  pull_request:
```

Comment:
It now runs twice each push if there is a PR open for the branch.

Reply:
```suggestion
on:
  push:
    branches:
    - master
  pull_request:
```

Reply:
There's still a small difference between the two currently "_duplicate_" jobs. Actions will build for the current branch and the other job will be built with the current branch merged into master.

Reply:
Even then, do you really need to do it twice? Jonpas' suggestion is perfect, no need to run it on every push. It only really becomes important whether it builds when it is merged into master, until then it is only secondary.